xwiki-rendering-syntaxes/xwiki-rendering-syntax-annotatedxhtml/src/main/java/org/xwiki/rendering/internal/renderer/xhtml/AnnotatedXHTMLChainingRenderer.java

```diff
@@ -83,4 +87,25 @@ public void endMacroMarker(String name, Map<String, String> parameters, String c
             this.macroRenderer.endRender(getXHTMLWikiPrinter());
         }
     }
+
+    @Override
+    public void beginMetaData(MetaData metadata)
+    {
+
+        Map<String, String> parameters = new HashMap<>();
```

Comment:
I would call them ``attributes`` because that's how they are referred to in the XML / HTML world.

Reply:
Indeed